modules/ROOT/pages/values-and-types/spatial.adoc

```diff
@@ -17,18 +17,17 @@ For more information about the comparison and ordering of spatial values, see th
 ====
 
 [[spatial-values-point-type]]
-== The Point type
+== The POINT type
 
-Neo4j supports the `Point` type for values of spatial geometry.
+Neo4j supports the `POINT` type for values of spatial geometry.
 
-Values with the `Point` type have the following characteristics:
+Values with the `POINT` type have the following characteristics:
 
 * Each point can have either 2 or 3 dimensions. 
 This means it contains either 2 or 3 64-bit floating point values, which together are called the _Coordinate_.
-* Each point will also be associated with a specific xref::values-and-types/spatial.adoc#cypher-spatial-crs[Coordinate Reference System] (CRS) that determines the meaning of the values in the _Coordinate_.
-* Instances of `Point` and lists of `Point` can be assigned to node and relationship properties.
-* Nodes and relationships with `Point` or `List(Point)` properties can be indexed using a xref::indexes-for-search-performance.adoc#indexes-create-a-node-point-index
-[point index]. 
+* Each point will also be associated with a specific xref::values-and-types/spatial.adoc#spatial-values-crs[Coordinate Reference System] (CRS) that determines the meaning of the values in the _Coordinate_.
+* Instances of `POINT` and `LIST<POINT>` can be assigned to node and relationship properties.
```

Comment:
I guess this is pending the decision on nullable inner types for constraints? as LIST<POINT> isn't technically assignable to a node, it should be LIST<POINT NOT NULL>

Reply:
in this case I guess `LIST<POINT NOT NULL>`, but I don't think we need to distinguish it here? like sure you cant store lists with nulls but this is mostly just pointing out that you can have lists of points or points in your properties (we just updated the type names). and until we introduce NOT NULL for types I don't think we should add it here.